denops/aider/bufferOperation.ts

```diff
@@ -278,19 +278,34 @@ async function openFloatingWindow(denops: Denops, bufnr: number): Promise<void>
   const terminal_height = Math.floor(ensure(await n.nvim_get_option(denops, "lines"), is.Number));
   const floatWinHeight = maybe(await v.g.get(denops, "aider_floatwin_height"), is.Number) || 20;
   const floatWinWidth = maybe(await v.g.get(denops, "aider_floatwin_width"), is.Number) || 100;
+  const floatWinStyle = maybe(await v.g.get(denops, "aider_floatwin_style"), is.String) || "minimal";
+  const floatWinBorder = maybe(await v.g.get(denops, "aider_floatwin_border"), is.String) || "double";
```

Comment:
https://neovim.io/doc/user/api.html#api-win_config
can you prohibit invalid values of these options by `is.LiteralOf` and `is.UnionOf` or something?

Reply:
It would fix the type errors you might see now.

Reply:
I think floatWinStyle should be like this.
```ts
  const floatWinStyle = maybe(
    await v.g.get(denops, "aider_floatwin_style"),
    is.UnionOf([is.LiteralOf("minimal"), is.Undefined]),
  );
```